templates/show/vpn/ipsec/connections/node.def

```diff
@@ -0,0 +1,6 @@
+help: Show VPN connections
+run: if pgrep charon >&/dev/null; then
+        sudo /usr/libexec/vyos/op_mode/show_ipsec_connections.py
```

Comment:
`${vyos_libexec_dir}` environment variable is present in 1.3, I think we should use it here.

Reply:
Fixed! `${vyos_op_scripts_dir}`